DSCResources/MSFT_xAzurePackAdmin/MSFT_xAzurePackAdmin.schema.mof

```diff
@@ -1,9 +1,10 @@
-[ClassVersion("1.0.0.0"), FriendlyName("xAzurePackAdmin")]
-class MSFT_xAzurePackAdmin : OMI_BaseResource
-{
-    [Write, Description("An enumerated value that describes if the principal is an Azure Pack admin.\nPresent {default}  \nAbsent   \n"), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
-    [Key, Description("The Azure Pack admin principal.")] String Principal;
-    [Required, EmbeddedInstance("MSFT_Credential"), Description("Credential to be used to perform the installation.")] String AzurePackAdminCredential;
-    [Required, Description("Database server for the Azure Pack databases.")] String SQLServer;
-    [Write, Description("Database instance for the Azure Pack databases.")] String SQLInstance;
-};
+[ClassVersion("1.0.0.0"), FriendlyName("xAzurePackAdmin")]
+class MSFT_xAzurePackAdmin : OMI_BaseResource
+{
+    [Write, Description("An enumerated value that describes if the principal is an Azure Pack admin.\nPresent {default}  \nAbsent   \n"), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Key, Description("The Azure Pack admin principal.")] String Principal;
+    [Required, EmbeddedInstance("MSFT_Credential"), Description("Credential to be used to perform the installation.")] String AzurePackAdminCredential;
+    [Required, Description("Database server for the Azure Pack databases.")] String SQLServer;

```

Comment:
Couldn't the same principle be an admin to multiple databases?  If so, we have not correctly identified the key.